aspnet/fundamentals/localization.rst

```diff
@@ -1,8 +1,302 @@
-.. include:: /../common/stub-topic.txt
+ASP.NET Core Internationalization, Localization and Globalization 
+==================================================================
 
-|stub-icon| Localization
-========================
+By `Rick Anderson`_ with assistance from `Damien Bowden <https://twitter.com/damien_bod>`_, `Bart Calixto @bartmax <https://twitter.com/bartmax>`_, and `Nadeem Afana <https://twitter.com/NadeemAfana>`__ 
 
-.. include:: /../common/stub-notice.txt
+Creating a multilingual website withe ASP.NET Core is not easy but it will allow your site to reach a wider audience. ASP.NET  has :doc:`/fundamentals/request-features` that provide support for localizing into different languages and cultures.
```

Comment:
Yes, the point should be internationalization is not easy. ASP.NET Core have a bunch of features that makes it easy.